content/en/docs/contributing/pull-requests.md

```diff
@@ -68,14 +68,14 @@ A([fa:fa-user New<br>Contributor]) --- id1[(open-telemetry/opentelemetry.io<br>G
 subgraph tasks[Changes using GitHub]
 direction TB
     0[ ] -.-
-    1[1. Edit this page] --> 2[2. Use GitHub markdown<br>editor to make changes]
-    2 --> 3[3. fill in Propose file change]
+    1[1\. Edit this page] --> 2[2\. Use GitHub markdown<br>editor to make changes]
+    2 --> 3[3\. fill in Propose file change]
```

Comment:
```suggestion
    1[1\. Edit this page] --> 2[2\. Use GitHub markdown<br>editor to make changes]
    2 --> 3[3\. Fill in Propose file change]
```

Reply:
Thank you.
I changed these characters in a bellow commit.

https://github.com/open-telemetry/opentelemetry.io/pull/6253/commits/c532ee535dcccac4ff377b347c55d3fd0c72484e